gh-clean-notifications

```diff
@@ -157,8 +156,14 @@ process_notifications() {
       echo "Marking as Done: $(echo "${item}" | jq -r .title)"
       echo "                 https://github.com$(echo "${item}" | jq -r .path)"
       IDS+=(-f "ids[]=$(echo "${item}" | jq -r .id)")
+      if test ${#IDS[@]} -eq 100
+      then
+        gh api graphql --silent "${IDS[@]}" -f query="${MARK_IDS_AS_DONE_QUERY}"
+	IDS=()
+      fi
```

Comment:
In the comment you wrote that it was saying that it was a maximum of 50 that could be archived at once, and here is testing that the length is equal to 100. Should this be updated to slice the array into the first 50, and then carry the leftovers into the next ID loop?

Reply:
The 100 is actually correct: `IDS` is an array that contains a pair of elements for each id: `-f` and `ids[]=<id>`. That explains why the array has length 100 when we're about to mark 50 notifications as done...

Reply:
Yeah, my bad forgot that it did that. This lgtm, and thank you for fixing =]